checkstyle/import-control.xml

```diff
@@ -27,6 +27,7 @@
   <allow pkg="javax.management" />
   <allow pkg="org.slf4j" />
   <allow pkg="org.junit" />
+  <allow pkg="org.hamcrest" />
```

Comment:
I'm OK with it although @junrao seemed less convinced last time the subject came up.